### ISSUE TITLE
README.md under Features, "Support for Android 4.0+" is false

### ISSUE THREAD

**Issue by K4sum1** [QuantumBadger/RedReader#1139]
In build.gradle.kts, minSdk is set to 16, which is Android 4.1. Android 4.0 would be SDK 14. Also according to F-Droid the minimum is Android 4.1 as well.

**Comment by QuantumBadger** [comment]
Thanks, I'll fix this. I'm planning to deprecate Android 4.x support in general soon due to a growing list of compatibility issues, and so in a couple of versions time Android 5.0 will be the minimum.

**Comment by K4sum1** [comment]
I switched to this client and have been awaiting Lemmy support (so I can switch away from Voyager) because it's the only client that I can find that works on Android 4.x. I like using apps with the oldest possible version compatibility, even on devices that don't need it.

**Comment by QuantumBadger** [comment]
Yeah, I've kept compatibility going as long as possible, but there are now a growing list of dependencies (OkHTTP, Jetpack Compose, etc) which only support API 21 (Android 5.0) and above. Users on Android 4.x will be able to continue using older versions, although I wouldn't recommend it due to security issues in the platform TLS implementation.
